blocks/commerce-b2b-po-require-approval-purchase-orders/commerce-b2b-po-require-approval-purchase-orders.js

```diff
@@ -0,0 +1,81 @@
+import { render as purchaseOrderRenderer } from '@dropins/storefront-purchase-order/render.js';
+import { RequireApprovalPurchaseOrders } from '@dropins/storefront-purchase-order/containers/RequireApprovalPurchaseOrders.js';
+import { PO_PERMISSIONS } from '@dropins/storefront-purchase-order/api.js';
+import { events } from '@dropins/tools/event-bus.js';
+import {
+  checkIsAuthenticated,
+  CUSTOMER_LOGIN_PATH,
+  CUSTOMER_PATH,
+  rootLink,
+} from '../../scripts/commerce.js';
+
+// Initialize
+import '../../scripts/initializers/purchase-order.js';
+
+const redirectToLogin = () => {
+  window.location.href = rootLink(CUSTOMER_LOGIN_PATH);
+};
+
+const redirectToAccountDashboard = () => {
+  window.location.href = rootLink(CUSTOMER_PATH);
```

Comment:
CUSTOMER_PATH page does not exist, maybe CUSTOMER_ACCOUNT_PATH
```suggestion
  window.location.href = rootLink(CUSTOMER_ACCOUNT_PATH);
```